sdk/python/jobs/automl-standalone-jobs/automl-forecasting-github-dau/auto-ml-forecasting-github-dau.ipynb

```diff
@@ -0,0 +1,841 @@
+    "\n",
+    "    pip install azureml-mlflow\n",
+    "\n",
+    "    pip install mlflow"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Obtain the tracking URI for MLFlow"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import mlflow\n",
+    "\n",
+    "# Obtain the tracking URL from MLClient\n",
+    "MLFLOW_TRACKING_URI = ml_client.workspaces.get(\n",
+    "    name=ml_client.workspace_name\n",
+    ").mlflow_tracking_uri\n",
+    "\n",
+    "print(MLFLOW_TRACKING_URI)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set the MLFLOW TRACKING URI\n",
+    "\n",
+    "mlflow.set_tracking_uri(MLFLOW_TRACKING_URI)\n",
+    "\n",
+    "print(\"\\nCurrent tracking uri: {}\".format(mlflow.get_tracking_uri()))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from mlflow.tracking.client import MlflowClient\n",
+    "\n",
+    "# Initialize MLFlow client\n",
+    "mlflow_client = MlflowClient()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Get the AutoML parent Job"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job_name = returned_job.name\n",
+    "\n",
+    "# Example if providing an specific Job name/ID\n",
+    "# job_name = \"funny_soursop_2zpkp35pdy\"\n",
+    "\n",
+    "# Get the parent run\n",
+    "mlflow_parent_run = mlflow_client.get_run(job_name)\n",
+    "\n",
+    "print(\"Parent Run: \")\n",
+    "print(mlflow_parent_run)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Print parent run tags. 'automl_best_child_run_id' tag should be there.\n",
+    "print(mlflow_parent_run.data.tags)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4.2 Get the AutoML best child run"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get the best model's child run\n",
+    "\n",
+    "best_child_run_id = mlflow_parent_run.data.tags[\"automl_best_child_run_id\"]\n",
+    "print(\"Found best child run id: \", best_child_run_id)\n",
+    "\n",
+    "best_run = mlflow_client.get_run(best_child_run_id)\n",
+    "\n",
+    "print(\"Best child run: \")\n",
+    "print(best_run)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "pd.DataFrame(best_run.data.metrics, index=[0]).T"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4.3 Download the best model locally\n",
```

Comment:
Please change the section headers including section numbers to match the energy demand notebook.